spec/models/course_spec.rb

```diff
@@ -39,7 +39,7 @@
       timestamp = 1.hour.ago
       course.provider.update updated_at: timestamp
       course.touch
-      expect(course.provider.updated_at).to eq timestamp
+      expect(course.provider.updated_at).to be_within(1.second).of timestamp
```

Comment:
Probably should use timecop

Reply:
I think the issue here is precision. Ruby time is a little more precise than what's saved in the database. The assertions were off by nanoseconds and failing in CI. See here for reference: https://tddium.wordpress.com/2011/08/07/rails-time-comparisons-devil-details-etc/

Reply:
I think the issue here might be precision. Ruby time seems to be a little more precise than what's saved in the database. The assertions were off by nanoseconds and failing in CI. See here for example: https://tddium.wordpress.com/2011/08/07/rails-time-comparisons-devil-details-etc/